.github/workflows/azure-fd-blob-fn-app.yml

```diff
@@ -0,0 +1,128 @@
+name: Azure Static Blob and Function CI/CD
+
+on:
+  workflow_dispatch:
```

Comment:
is this meant to have a sub property? Or is it grouped with the following `push:` one?

Reply:
we are not setting any of the sub properties and just using the default settings, I just used the example syntax that I found and ran the linter to confirm it passed. It could be bad form tho. @Ethan-Arrowood, do you have a recommendation?

Reply:
I'm not familiar enough with actions to know for sure

Reply:
I'm going to leave this as it is then.